pkg/config/config.go

```diff
@@ -439,6 +440,12 @@ func UseViper(v *viper.Viper) error {
 		return err
 	}
 
+	tmp := v.Get("clouderies")
```

Comment:
Can you also update the example config file? https://github.com/cozy/cozy-stack/blob/master/cozy.example.yaml

Reply:
Done